packages/ui/src/layout/TopBar.tsx

```diff
@@ -40,13 +68,50 @@ export const TopBar: FunctionComponent<ITopBar> = (props) => {
 
       <MastheadContent>
         <ToolbarItem className="shell__link">
-          <Link to={ExternalLinks.GitHub} target="_blank" rel="noopener noreferrer">
-            <Button variant="plain" aria-label="edit">
-              <Icon size="lg">
-                <GithubIcon />
-              </Icon>
-            </Button>
-          </Link>
+          <Dropdown
+            onSelect={onSelect}
+            toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+              <MenuToggle ref={toggleRef} onClick={onToggle} variant="plain" isExpanded={isOpen}>
+                <EllipsisVIcon />
+              </MenuToggle>
+            )}
+            isOpen={isOpen}
+            onOpenChange={(isOpen: boolean) => setIsOpen(isOpen)}
+            popperProps={DEFAULT_POPPER_PROPS}
+          >
+            <a href="https://kaoto.io/workshop/" target="_blank">
+              <DropdownItem key="tutorial">
+                <Icon isInline>
+                  <ExternalLinkAltIcon />
+                </Icon>
+                &nbsp;<span className="pf-u-mr-lg">Tutorials</span>
+              </DropdownItem>
+            </a>
+            <a href="https://kaoto.io/docs/" target="_blank">
+              <DropdownItem key="help">
+                <Icon isInline>
+                  <ExternalLinkAltIcon />
+                </Icon>
+                &nbsp;<span className="pf-u-mr-lg">Help</span>
+              </DropdownItem>
+            </a>
+            <a href="https://github.com/KaotoIO/kaoto-next/issues/new/choose" target="_blank">
```

Comment:
Note: we're wrapping the `<DropdownItem />` component to make the entire button area clickable for the link